src/main/java/com/github/fishio/settings/Settings.java

```diff
@@ -0,0 +1,373 @@
+	 * 		setting to get the value for.
+	 * @return
+	 * 		a SimpleDoubleProperty with the value of the setting or NaN if the setting is not found.
+	 */
+	public SimpleDoubleProperty getDoubleProperty(String setting) {
+		SimpleDoubleProperty s = doubleSettings.get(setting);
+		if (s == null) {
+			log.log(LogLevel.ERROR, "Setting '" + setting + "' not found!");
+			return new SimpleDoubleProperty(Double.NaN);
+		}		
+		return s;
+	}
+	
+	/**
+	 * Get the Integer Property of the specified setting.
+	 * @param setting
+	 * 		setting to get the value for.
+	 * @return
+	 * 		a Property<Number> with the value of the setting or Integer.MIN_VALUE if the setting is not found.
+	 */
+	public SimpleIntegerProperty getIntegerProperty(String setting) {
+		SimpleIntegerProperty s = integerSettings.get(setting);
+		if (s == null) {
+			log.log(LogLevel.ERROR, "Setting '" + setting + "' not found!");
+			return new SimpleIntegerProperty(Integer.MIN_VALUE);
+		}		
+		return s;
+	}
+	
+	/**
+	 * Get the Boolean Property of the specified setting.
+	 * @param setting
+	 * 		setting to get the value for.
+	 * @return
+	 * 		a Boolean eProperty with the value of the setting.
+	 */
+	public SimpleBooleanProperty getBooleanProperty(String setting) {
+		SimpleBooleanProperty s = booleanSettings.get(setting);
+		if (s == null) {
+			log.log(LogLevel.ERROR, "Setting '" + setting + "' not found!");
+			return new SimpleBooleanProperty();
+		}
+		return s;
+	}
+	
+	/**
+	 * Get the value of the specified setting.
+	 * @param setting
+	 * 		The setting to get the value of.
+	 * @return
+	 * 		The value of the setting or NaN if the setting is not found.
+	 */
+	public double getDouble(String setting) {
+		return getDoubleProperty(setting).getValue().doubleValue();
+	}
+	
+	/**
+	 * Get the value of the specified setting.
+	 * @param setting
+	 * 		The setting to get the value of.
+	 * @return
+	 * 		The value of the setting.
+	 */
+	public KeyCode getKeyCode(String setting) {
+		return keySettings.get(setting);
+	}
+	
+	/**
+	 * Get the value of the specified setting.
+	 * @param setting
+	 * 		The setting to get the value of.
+	 * @return
+	 * 		The value of the setting or NaN if the setting is not found.
+	 */
+	public int getInteger(String setting) {
+		return getIntegerProperty(setting).getValue().intValue();
+	}
+	
+	/**
+	 * Get the value of the specified setting.
+	 * @param setting
+	 * 		The setting to get the value of.
+	 * @return
+	 * 		The value of the setting or NaN if the setting is not found.
+	 */
+	public boolean getBoolean(String setting) {
+		return getBooleanProperty(setting).getValue().booleanValue();
+	}
+	
+	/**
+	 * Set the value of a setting.
+	 * @param setting
+	 * 		the setting to set the new value for.
+	 * @param newValue
+	 * 		The new value for the setting.
+	 */
+	public void setDouble(String setting, double newValue) {
+		getDoubleProperty(setting).setValue(newValue);
+		log.log(LogLevel.DEBUG, setting + " changed to " + newValue);
```

Comment:
*Auto-generated comment*

    
    The String literal " changed to " appears 4 times in this file; the first occurrence is on line 217